### PR TITLE
feat: add logging configurability to the simulator

### LINKED ISSUE #377: feat: Add logging configurability to the simulator
As a Simulator operator
I want to be able to see detailed logging from the gRPC and Netty connection layers as both and `producer` and `consumer`
So that I can more easily debug gRPC and HTTP/2 connection issues between the Simulator and the Block Node

## Tech Notes
* If possible, make configuration of the Simulator logging similar to the Block Node with a single `logging.properties` file that's used to enable and specify the logging levels for the BN app as well as Helidon

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR adds logging configuration in the simulator, this way using logging.properties one can set the verbose

**Related issue(s)**:

Fixes #377 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by mattp-swirldslabs — APPROVED** [review]
Looks good

**Review by ata-nas — APPROVED** [review]
Looks good, seems like `codecov` is preventing your merge though.

File: simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java [hunk 1]
```diff
@@ -128,4 +137,24 @@ public StreamStatus getStreamStatus() {
                 .lastKnownPublisherStatuses(publishStreamGrpcClient.getLastKnownStatuses())
                 .build();
     }
+
+    private void loadLoggingProperties() {
+        final LogManager logManager = LogManager.getLogManager();
+        try (InputStream is = BlockStreamSimulator.class.getClassLoader().getResourceAsStream(LOGGING_PROPERTIES)) {
+            logManager.readConfiguration(is);
+        } catch (IOException | NullPointerException e) {
```
Comment (AlfredoG87) [inline simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java:145]: Where could the NPE come from?

**Review by AlfredoG87 — APPROVED** [review]
LG, just a nit question.

File: simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java (hunk 1 above)
Comment (AlfredoG87) [reply simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java:145]: ohh I see the LogManager could be null, haha. just answered myself here :)

**Review by ata-nas — APPROVED** [review]

**Pull request merged**